indra/newview/character/avatar_constraint.llsd

```diff
@@ -11,30 +11,30 @@
         },
     'mCollarLeft':
         {
-            'cone_angle':r9,
+            'cone_angle':r3,
             'forward_axis':
                 [r-0.021,r0.085,r0.165],
-            'max_twist':r18,
-            'min_twist':r-18,
+            'max_twist':r3,
+            'min_twist':r-3,
```

Comment:
I'm tightening the collar constraints to keep them from breaking out for some difficult target positions.